r/PACKAGING.md

```diff
@@ -0,0 +1,129 @@
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Packaging checklist for CRAN release
+
+For a high-level overview of the release process see the
+[Apache Arrow Release Management Guide](https://arrow.apache.org/docs/developers/release.html#post-release-tasks).
+
+Before the release candidate is cut:
+
+- [ ] [Create a GitHub issue](https://github.com/apache/arrow/issues/new/)
+  entitled `[R] CRAN packaging checklist for version X.X.X`
+  and copy this checklist to the issue.
+- [ ] Evaluate the status of any failing
+  [nightly tests and nightly packaging builds](https://lists.apache.org/list.html?builds@arrow.apache.org). These checks
+  replicate most of the checks that CRAN runs, so we need them all to be passing
+  or to understand that the failures may (though won't necessarily) result in a rejection from CRAN.
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
+- [ ] Ensure the contents of the README is accurate
+- [ ] Run `urlchecker::url_check()` on the R directory at the release candidate
+  commit. Ignore any errors with badges as they will be removed in the CRAN release branch.
+- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later).
+- [ ] For major releases, prepare tweet thread highlighting new features
+
+Wait for the release candidate to be cut:
+
+- [ ] Release candidate!
+
+Make pull requests into the [autobrew](https://github.com/autobrew) and
+[rtools-packages](https://github.com/r-windows/rtools-packages) repositories
+used by the configure script on MacOS and Windows. These pull requests will
+use the release candidate as the source.
+
+- [ ] Pull request to modify 
+  [the apache-arrow autobrew formula]( https://github.com/autobrew/homebrew-core/blob/master/Formula/apache-arrow.rb) 
+  to update the version, SHA, and any changes to dependencies and build steps that have changed in the
```

Comment:
Where do I get the SHA?

Reply:
Same question applies ;)

Reply:
If I google "how to get the SHA of a file", different results refer to SHA1 and SHA256; this bit needs explicit instructions as it's unclear as it is.

Reply:
Oh wait, it's SHA256, cool